x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/alerts_by_status.tsx

```diff
@@ -80,6 +81,13 @@ interface AlertsByStatusProps {
   signalIndexName: string | null;
 }
 
+enum Statuses {
+  OPEN = 'open',
+  CLOSED = 'closed',
+  ACKNOWLEDGED = 'acknowledged',
+}
```

Comment:
don't we already have this constant defined somewhere?

Reply:
found it in common types 👌